app/app.go

```diff
@@ -370,6 +356,20 @@ func NewSommelierApp(
 			app.CorkKeeper.Hooks(),
 		))
 
+	// register the proposal types
```

Comment:
I'm curious, how did moving the addition of governance routes help?

Reply:
The govRouter needed to be instantiated *after* the keepers, otherwise the proposals would have nil panics